### PR TITLE
746 add button for remove sanger, splitted add/view/update commands into separate files

### PR DESCRIPTION
**Pull request by patrikgrenfeldt**

### CODE REVIEW

**Comment by patrikgrenfeldt** [comment]
@moonso we added tests for sanger/cancel sanger, anything else?

File: tests/server/blueprints/variants/conftest.py [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+from scout.server.app import create_app
+from scout.adapter import MongoAdapter
+from scout.load.hgnc_gene import load_hgnc_genes
+from scout.load.hpo import load_hpo
+
+
+log = logging.getLogger(__name__)
+
+
+class MockMail:
```
Comment (moonso) [inline tests/server/blueprints/variants/conftest.py]: 👍

File: scout/adapter/mongo/hgnc.py [hunk 2]
```diff
@@ -14,10 +14,10 @@ def load_hgnc_gene(self, gene_obj):
             gene_obj(dict)
 
         """
-        LOG.debug("Loading gene %s, build %s into database" %
-                     (gene_obj['hgnc_symbol'], gene_obj['build']))
+        #LOG.debug("Loading gene %s, build %s into database" %
```
Comment (moonso) [inline scout/adapter/mongo/hgnc.py]: Why is this commented?

**Comment by moonso** [comment]
Great work @patrikgrenfeldt @emmser !!

**Pull request merged**